core/arch/arm/mm/core_mmu.c

```diff
@@ -967,7 +967,9 @@ static void check_pa_matches_va(void *va, paddr_t pa)
 			panic("issue in linear address space");
 		return;
 	}
-	if (v >= (CFG_TEE_LOAD_ADDR & ~CORE_MMU_PGDIR_MASK) &&
+	/* wait pager is initialize before looking into pager data */
+	if (cpu_mmu_enabled() &&
```

Comment:
OK, we have the same problem on qemu v7 with pager enabled and `CFG_TEE_CORE_DEBUG=y`.
The problem is that  `core_mmu_linear_map_end` hasn't been updated yet. Either we update it earlier or we replace that variable with something else.